scripts/tools/type_checker.sh

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env bash
 
-pyright --stats
```

Comment:
the CI is using `scripts/ci/style/pyright_check.sh`, running in the checker docker image, not this file

Reply:
yeah modifying this for local test